grade-school/GradeSchoolTest.swift

```diff
@@ -2,82 +2,89 @@ import XCTest
 
 // Apple Swift version 2.0
 
-class GradeSchoolTest: XCTestCase
-{
-
+class GradeSchoolTest: XCTestCase {
     func testAnEmptySchool() {
-        let school = GradeSchool()
+        let school   = GradeSchool()
         let expected = [:]
-        let result = school.db
+        let result   = school.roster
         XCTAssertEqual(result, expected)
     }
-    
+
     func testAddStudent() {
-        let school = GradeSchool()
+        var school = GradeSchool()
         school.addStudent("Aimee", grade: 2)
-        let result = school.db
+        let result = school.roster
         let expected: Dictionary = [2: ["Aimee"]]
         XCTAssertEqual(Array(result.keys), Array(expected.keys))
-        XCTAssertEqual(result[2]!, expected[2]!)
```

Comment:
I think assert equal is the way to go here but I do agree that we should not force unwrap these optionals.  There was an issue early in xctesf where xct assert equals did not work on optionals but I believe now it does.